Softeq.XToolkit.WhiteLabel.Forms/Navigation/FormsViewLocator.cs

```diff
@@ -24,6 +24,11 @@ public class FormsViewLocator : IFormsViewLocator
                 {
                     return FindNavigationForViewModel(masterDetailPage.Detail.Navigation, viewModel);
                 }
+
+                if (page is TabbedPage tabbedPage)
```

Comment:
Optional: `switch` + type matching can be used here (like in `SetupPage` method below)